lib/fog/openstack/models/collection.rb

```diff
@@ -0,0 +1,47 @@
+require 'fog/core/collection'
+
+module Fog
+  module OpenStack
+    class Collection < Fog::Collection
+      # It's important to store the whole response, it contains e.g. important info about whether there is another
+      # page of data.
+      attr_accessor :response
+      # In some cases it's handy to be able to store the tenant for the record, e.g. swift doesn't contain tenant info
+      # in the result, so we can track it in this attribute based on what tenant was used in the request
+      attr_accessor :tenant
```

Comment:
ah, right, keystone v3 is using project. Will do, though I placed this badly, I need it in Fog::Model actually, to track project per record